cantor-archive/src/main/java/com/salesforce/cantor/archive/AbstractBaseArchiver.java

```diff
@@ -25,6 +27,11 @@
 import static com.salesforce.cantor.common.CommonPreconditions.checkString;
 
 abstract class AbstractBaseArchiver {
+    private static final String archivePrefixFormat = "archive-%s-%s-";
+
+    static Path createTempFile(final String namespace) throws IOException {
+        return File.createTempFile(String.format(archivePrefixFormat, CantorProperties.getKingdom(), namespace), null).toPath();
```

Comment:
@AlexanderKouthoofd `namespace` is the unique identifier for an installation of cantor, let's just use that. And remove `getKingdom` method - sfdc specific.

Reply:
Yeah good point, wasn't really thinking about that. Removed.